### PR TITLE
DEV-5475 - Updated Alpha Bank payment gateway

### PR DESCRIPTION
**Pull request by mfutselaar**
[DEV-5475]

[DEV-5475]: https://enartia.atlassian.net/browse/DEV-5475?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ggeorg0p** [comment]
@mfutselaar is it possible to create a diff file in order to check changes?

**Comment by mfutselaar** [comment]
> @mfutselaar is it possible to create a diff file in order to check changes?

I was already working on that.

It's not really a diff, it took me a while to figure out how to do it and it turned out to be just "add a function". The function is available [here (line 310:328](https://github.com/Enartia/wordpress-plugins/blob/f0d37b8d637e932f9d150f7d67a3ce8b188647c8/woo-alpha-bank-payment-gateway/wooshop-alpha.php#L310)

Aside from that the logo file has changed, the file reference has also been changed [here](https://github.com/Enartia/wordpress-plugins/blob/f0d37b8d637e932f9d150f7d67a3ce8b188647c8/woo-alpha-bank-payment-gateway/wooshop-alpha.php#87) (line 87).

File: woo-alpha-bank-payment-gateway/README.txt [hunk 1]
```diff
@@ -0,0 +1,85 @@
+=== Payment Gateway - Alpha Bank for WooCommerce ===
+Contributors: enartia,georgekapsalakis,g.georgopoulos,akatopodis
+Author URI: https://www.papaki.com	
+Tags: ecommerce, woocommerce, payment gateway, alphabank, alpha
+Tested up to: 5.9.0
+Requires at least: 4.0
+Stable tag: 1.3.7
```
Comment (ggeorg0p) [inline woo-alpha-bank-payment-gateway/README.txt:7]: You should increase  the version to something like 1.4.0 probably

Comment (ggeorg0p) [reply woo-alpha-bank-payment-gateway/README.txt:7]: REsolved in Slack https://teamdotblue.slack.com/archives/D05Q6HH7T8S/p1709040968816519

**Review by ggeorg0p — APPROVED** [review]

**Pull request closed without merging**